openshift/e2e-common.sh

```diff
@@ -191,6 +191,39 @@ function run_e2e_tests(){
   local test_name=$1 
   local failed=0
 
+  # Keep this in sync with test/ha/ha.go
+  readonly REPLICAS=2
+  # TODO: Increase BUCKETS size more than 1 when operator supports configmap/config-leader-election setting.
+  readonly BUCKETS=1
+
+  # Keep the bucket count in sync with test/ha/ha.go
+  # TODO: configure it in KnativeServing when operator supports it.
```

Comment:
Do you want to keep this long comment here? Otherwise looks good.

Reply:
Let's remove it :smile:  Done.